src/mount/readdata.cc

```diff
@@ -753,6 +756,14 @@ int read_to_buffer(ReadRecord *rrec, uint64_t current_offset,
 			try_counter = 0;
 		} catch (UnrecoverableReadException &ex) {
 			print_error_msg(reader, try_counter, ex);
+			if (gShowMessagesOnNotificationArea) {
+				if (errorInodeFullPath.empty()) {
+					fs_fullpath(rrec->inode, 0, 0, errorInodeFullPath);
+				}
+				ShowMessageOnNotificationArea(
+				    "Unrecoverable read error: " + std::string(ex.what()) +
+				    " " + errorInodeFullPath.c_str());
```

Comment:
The function `c_str()` is redundant in this case:
```suggestion
				    " " + errorInodeFullPath);
```